files/files.go

```diff
@@ -48,17 +48,28 @@ func Write(output string, fileName string, content string) {
 	}
 
 	// set permissions on file and folder
+	fmt.Println("owner", owner)
+	if owner != nil {
+		fmt.Println("owner is set")
+		setPermissions(f, path, output, *owner)
+		return
+	}
 	if config.Config.Owner != -1 {
+		fmt.Println("config.Config.Owner is set")
```

Comment:
i was about to say that this should not be there.
But i guess it is okay to have some output ?

Reply:
naah you are right - that was jus there to debug :D